tools/importer/import.js

```diff
@@ -350,6 +350,12 @@ function customLogic(main, doc, url) {
   // Add downloads block
   if (doc.querySelector('.downloads-wrapper')) {
     const cells = [['press-cards']];
+    let styleTable;
+    if (doc.querySelectorAll('.download-item--landscape').length > 0) {
+      const styleCells = [['Section Metadata']];
```

Comment:
I am wondering if this should be a style attribute of the `press-cards` block instead of `Section Metadata` block. Reason being this style would only be relevant for sections which have a `press-block`.

Reply:
Then we wouldn't modify the importer, right ?

Reply:
I think you would still need to do that here for `landscape` , something like 
```
cells = [['press-cards(landscape)']];
```